source/sitemap.xml

```diff
@@ -31,7 +31,7 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>http://developers.virtusize.com/integration-guide/integration.html</loc>
```

Comment:
No, it was a mistake done when I added `mobile`... I'll bring it back :stuck_out_tongue_winking_eye: